src/core/layertree/qgslayertreemodel.h

```diff
@@ -233,25 +234,45 @@ class CORE_EXPORT QgsLayerTreeModel : public QAbstractItemModel
      * Force only display of legend nodes which are valid for given map settings.
      * Setting NULLPTR or invalid map settings will disable the functionality.
      * Ownership of map settings pointer does not change, a copy is made.
-     * \since QGIS 2.6
```

Comment:
The deletion of \since is something we do all the time when we add a \deprecated tag? Wondering what's best practice here considering that function will still be the only one available for the current 3.28 LTR.

Reply:
Yeah python docstrings don't permit both since and deprecated - it's one or the other.i think for 3.28 a plugin dev should be consulting the 3.28 API docs, not a later version